package.json

```diff
@@ -39,6 +39,7 @@
     "cover": "ocular-test cover",
     "lint": "ocular-lint",
     "publish": "ocular-publish",
+    "version": "ocular-build webgl",
```

Comment:
What does this do and why do we need/want it?

Reply:
It re-transpiles the webgl module after version bump to update the inlined version number.